js/pagedtable.js

```diff
@@ -576,16 +620,19 @@ var PagedTable = function (pagedTable, source) {
     };
 
     me.calculateWidths = function(measures) {
+      // --> Here is where we want to precompute the html columns.
```

Comment:
I'd remove all the `-->` from the comments and keep all comments with the same formatting.